actool/manifest.go

```diff
@@ -250,6 +264,31 @@ func patchManifest(im *schema.ImageManifest) error {
 		}
 	}
 
+	if patchSeccompMode != "" {
+		i := 0
```

Comment:
I don't think it's half-baked, it's just specifying the current (unfortunate) behaviour. How do you want to move forward here?